src/common/dao/user.go

```diff
@@ -258,9 +258,12 @@ func DeleteUser(userID int) error {
 }
 
 // ChangeUserProfile ...
-func ChangeUserProfile(user models.User) error {
+func ChangeUserProfile(user models.User, cols ...string) error {
```

Comment:
Could you update the comment to explain the usage of `cols`?

Reply:
OK, add comment to it